SS4 compatibility

Firesphere opened the pull request:
Unacceptable! Needs squashy!

Commit:
Add a usercount badge for readme's

Commit:
Set up SS4

Require guzzle

Clean up and refactor. Clean code. Be a tidy Dutchy

Badges badges badges badges, mushroom mushroom!

Firesphere commented:
Why is CodeClimate taking Framework in to account? :|

zanderwar commented on composer.json:
```diff
@@ -17,18 +17,25 @@
   },
-  "target-dir": "stripeslack"
+  "extra": {
+    "installer-name": "stripeslack"
```
You're missing `expose` here for your image directory

Firesphere replied:
Yep, good one, should be fixed by now :)

Commit:
Move to vendor module

Commit:
Readying SS4

Commit:
Update license

Commit:
Minor code updates

Commit:
A bit of cleaning

Commit:
PHP-CS-Fixer

Commit:
Name config

Firesphere committed:
Update licens for packagist

Commit:
FIX: Bind to package, not recipe

The recipe is one way of installing the CMS, but not the only way. This change also fixes situations where the silverstripe/recipe-cms ^4 has been installed, rather than ^1 (the numbering of which was a bug)

Commit:
FIX: Fix slack form action

The action needs to be public.

Firesphere committed:
Merge pull request #14 from sminnee/ss4

FIX: Bind to package, not recipe
